src/main/java/it/gov/pagopa/pu/organization/model/SilServiceAuthConfig.java

```diff
@@ -0,0 +1,6 @@
+package it.gov.pagopa.pu.organization.model;
+
+import java.io.Serializable;
+
+public interface SilServiceAuthConfig extends Serializable {
```

Comment:
let's use @JsonTypeInfo in order to configure the polimorphic choice which jackson should use to read it from db or REST API (see `
it/gov/pagopa/payhub/activities/dto/debtposition/syncwfconfig/WfExecutionConfig.java` for an example